contracts/eden/src/encrypt.cpp

```diff
@@ -0,0 +1,30 @@
+#include <encrypt.hpp>
+
+namespace eden
+{
+   encrypt::encrypt(eosio::name contract, eosio::name scope)
+       : contract(contract), encrypted_data_tb{contract, scope.value}
+   {
+   }
+
+   void encrypt::set(uint64_t id, const std::vector<encrypted_key>& keys, const eosio::bytes& data)
+   {
+      encrypted_data_tb.emplace(contract, [&](auto& row) {
```

Comment:
This doesn't allow replacing the data; e.g. when the meeting link went bad (very frequent with Zoom)